GeoJSON/specification.md

```diff
@@ -104,15 +104,92 @@ This section contains references to standards documents and related resources.
+| GM_Primitive      | not supported | `GM_Primitive` is an abstract type. |
+| GM_Ring           | Polygon | A `GM_Ring` is not intended to be used as a standalone geometry object. It is typically used to define an interior or exterior boundary in a Surface and is thus mapped to Polygon. |
+| GM_Surface        | Polygon | A `GM_Surface` can have many SurfacePatches, it is thus mapped to MultiPolygon. |
+| GM_Tin            | not supported | TODO A TIN without triangulation could be converted to a MultiPoint object. |
+| GM_Triangle       | Polygon |  |
+
+#### ISO 19108 - Temporal types
+
+For types from ISO 19108 used in INSPIRE schemas, suitable mappings need to be found on a case-by-case basis. The default should be to use the [Simple Period](../model-transformation/SimplePeriod.md) substitution rule.
+
+#### ISO 19115 - Metadata types
+
+For types from ISO 19108 used in INSPIRE schemas, suitable mappings need to be found on a case-by-case basis. For `CI_Citation`, the [Simple Citation](../model-transformation/SimpleCitation.md) substitution rule can be applied.
+
+#### Abstract Types and Inheritance
+
+As `abstract` types cannot be encoded directly, the inheritance hierarchy is collapsed to the concrete types. Abstract types are then removed from the model. 
+
+#### Union Types
 
-* DECIDE: Should namespace prefixes be retained for the JSON property names?
+A `union` represents a choice between multiple properties with different value types, such as a `ReferenceType` and a `BuildingType`. They can have a multiplicity other than exactly 1. The most common use case for union types in INSPIRE is to offer a choice between encoding an object in place or to encode it by reference.
 
-All model transformation rules are applied in such a way that the resulting proeprty names for valid XML element and type names, and are useable as property names in JSON.
+For the purpose of creating GeoJSON, `union` types are replaced with the concrete type in the `union`, unless the [Property Composition to Association transformation rule](../model-transformation/PropertyCompositionToAssociation.md) is applied by the theme profile.
+
+#### Enumerations and Code Lists
+
+Enumerations and classes with the sterotype `<<Codelist>>` remain unchanged.
+
+### Properties
+
+Property names remain as they are.
+
+* TODO: Should namespace prefixes be retained for the JSON property names?
+
+#### Properties with a `uom` attribute
+
+The unit of measurement attribute (`uom`) on any property `x` has to be retained. It is transformed to a new property of the type with the name `x.uom`.
+
+#### Arrays
+
+Property types for properties with a cardinality greater than `1` and a simple property type (e.g. String, Integer, Float, ...)may use arrays of these simple types. More information is available in the [Extract Primitive Array](../model-transformation/ExtractPrimitiveArray.md) transformation rule.
+
+#### Voidable
+
```

Comment:
But we should say (probably in the instance conversion rule) how to deal with properties for which no data is available.

Reply:
OK.